README.md

```diff
@@ -1 +1,101 @@
-# Dash Template
+### Interested in contributing a code example?
+
+Please take a look at our [contribution guidelines](CONTRIBUTING.md) before
+getting started. Thank you!
+
+The Dash template is intended for demos that would benefit from a user interface. This user interface could include settings to run and customize the problem, an interactive graphical element, or tables/charts to compare different solutions. This template is also useful for demos that are intended for a general audience, as it is more approachable for those without a technical background.
+
+<!-- Before submitting your code, please delete everything above and including this line. -->
+
+# Demo Name
+
+Describe your demo and specify what it is demonstrating. Consider the
+following questions:
+
+* Is it a canonical problem or a real-world application?
+* Does it belong to a particular domain such as material simulation or logistics?
+* What level of Ocean proficiency does it target: beginner, advanced?
+
+A clear description allows us to properly categorize your demo.
+
+Please include a screenshot of your demo below.
+
+![Demo Example](static/demo.png)
+
```

Comment:
I think there needs to be something here that lets the contributor know that the following is boilerplate because the preceding is an explanation of the template to the contributor and the following content is for the end user

Reply:
Added some comments, let me know if this works: https://github.com/dwave-examples/template-dash/pull/3/commits/69ce8de2d571b557b6076d01df9822abb529133b